packages/typespec-test/test/openai_modular/generated/typespec-ts/src/models/models.ts

```diff
@@ -257,7 +257,7 @@ export interface CompletionsLogProbabilityModel {
   /** The textual forms of tokens evaluated in this probability model. */
   tokens: string[];
   /** A collection of log probability values for the tokens in this completions data. */
-  tokenLogprobs: (number | null)[];
+  tokenLogprobs: (number | null)[] | null;
```

Comment:
Given this is how tokenLogprobs defined.
```
alias NullableFloat = float32 | null;  
tokenLogprobs: NullableFloat[];
```
I feel like the previous change is correct ?

Reply:
looks like you already changed.